dist/html.js

```diff
@@ -46,19 +53,19 @@ const Outline = ({
       className: "outline-topic-row"
     }, /*#__PURE__*/_react.default.createElement("td", null, /*#__PURE__*/_react.default.createElement("a", {
       href: `#${topic.topicId}`
-    }, i + 1, ". ", topic.topicName)), /*#__PURE__*/_react.default.createElement("td", null, topic.claimsCount), /*#__PURE__*/_react.default.createElement("td", null, (100 * topic.claimsCount / totalClaims).toFixed(0), "%")));
+    }, i + 1, ". ", topic.topicName)), /*#__PURE__*/_react.default.createElement("td", null, topic.claimsCount), SHOW_STATS && /*#__PURE__*/_react.default.createElement("td", null, (100 * topic.claimsCount / totalClaims).toFixed(0), "%")));
     topic.subtopics.forEach((subtopic, j) => {
       rows.push( /*#__PURE__*/_react.default.createElement("tr", {
         key: `${i}/${j}`,
         className: "outline-subtopic-row"
       }, /*#__PURE__*/_react.default.createElement("td", null, /*#__PURE__*/_react.default.createElement("a", {
         href: `#${subtopic.subtopicId}`
-      }, i + 1, ".", j + 1, ". ", subtopic.subtopicName)), /*#__PURE__*/_react.default.createElement("td", null, subtopic.claimsCount), /*#__PURE__*/_react.default.createElement("td", null, (100 * subtopic.claimsCount / totalClaims).toFixed(0), "%")));
+      }, i + 1, ".", j + 1, ". ", subtopic.subtopicName)), /*#__PURE__*/_react.default.createElement("td", null, subtopic.claimsCount), SHOW_STATS && /*#__PURE__*/_react.default.createElement("td", null, (100 * subtopic.claimsCount / totalClaims).toFixed(0), "%")));
     });
   });
   return /*#__PURE__*/_react.default.createElement("div", {
     id: "outline"
-  }, /*#__PURE__*/_react.default.createElement("table", null, /*#__PURE__*/_react.default.createElement("thead", null, /*#__PURE__*/_react.default.createElement("tr", null, /*#__PURE__*/_react.default.createElement("th", null, "Topic/Subtopic"), /*#__PURE__*/_react.default.createElement("th", null, "Claims"), /*#__PURE__*/_react.default.createElement("th", null, "%"))), /*#__PURE__*/_react.default.createElement("tbody", null, rows)));
+  }, /*#__PURE__*/_react.default.createElement("table", null, /*#__PURE__*/_react.default.createElement("thead", null, /*#__PURE__*/_react.default.createElement("tr", null, /*#__PURE__*/_react.default.createElement("th", null, "Topic/Subtopic"), /*#__PURE__*/_react.default.createElement("th", null, "Arguments"), SHOW_STATS && /*#__PURE__*/_react.default.createElement("th", null, "%"))), /*#__PURE__*/_react.default.createElement("tbody", null, rows)));
```

Comment:
This can be done in a follow up, but FYI was talking to Deger last week about how we use "claims" and "arguments" interchangeably for statements that are, arguably, not arguments :)

If you agree, at some point we should do a pass to change "aruguments" to "claims" wherever it appears.

This change can be pushed without addressing this comment